www/src/pages/en/usage/trpc.md

```diff
@@ -258,10 +258,13 @@ const MyComponent = () => {
     async onMutate(newPost) {
       // Cancel outgoing fetches (so they don't overwrite our optimistic update)
       await utils.post.list.cancel();
+
       // Get the data from the queryCache
       const prevData = utils.post.list.getData();
```

Comment:
Not sure about the API changes, Julius can probably give a comment on that. But if we're not using `prevData` anymore then we don't need it here.

Reply:
I let `prevData` like it is, as one can use it to put it into the context by returning it from `onMutate`. That way, you can use the previous data to roll back (see `onError` below) on a failed mutation.

Reply:
Oh, I just realized there's also another old `setData` call in `onError`. Will add another commit.